tests/testthat/test-build-home-authors.R

```diff
@@ -26,3 +26,14 @@ test_that("names can be removed from persons", {
   expect_length(remove_name(p4$comment, "ORCID"), 0)
   expect_length(remove_name(p5$comment, "ORCID"), 0)
 })
+
+test_that("Comments in authors info are linkified", {
+  p <- list(name = "Jane Doe",
```

Comment:
Can you please use standard tidyverse styling?

Reply:
Oops, now I think it looks like https://style.tidyverse.org/syntax.html#long-lines :-)